src/assets/styles/_colors.scss

```diff
@@ -2,8 +2,20 @@
   --primary-color: cadetblue;
   --secondary-color: #ff3d00;
   --primary-dark-color: black;
-  --primary-header-color: grey;
-  --foreground-rgb: rgb(0 0 0);
-  --background-start-rgb: rgb(214 219 220);
-  --background-end-rgb: rgb(255 255 255);
+  --secondary-dark-color: #e6e6e6;
+  --primary-box-shadow: #eaeaea;
+  --primary-background-header: hsla(0, 0%, 100%, 0.8);
+  --primary-background-color: white;
+  --primary-active-color: hsla(0, 0%, 16%);
+}
+
+:root.dark {
+  --primary-color: cadetblue;
+  --secondary-color: #ff3d00;
+  --primary-dark-color: white;
+  --secondary-dark-color: #383838;
+  --primary-box-shadow: #343434;
+  --primary-background-header: hsla(0, 0%, 17%, 0.8);
+  --primary-background-color: rgb(35 39 47/1);
+  --primary-active-color: hsla(0, 0%, 16%);
 }
```

Comment:
Please add media query `prefers-color-scheme` to get browser’s color scheme, and then when the user will click on the theme toggle you can add a class to the html to change the color scheme